test/e2e-go/e2e_go_olm_test.go

```diff
@@ -69,3 +87,75 @@ var _ = Describe("Integrating Go Projects with OLM", func() {
 		})
 	})
 })
+
+const uncommentedCertManager = `# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'.
```

Comment:
So the variable names are the same
```suggestion
const uncommentedCertManagerKustomizeFields = `# [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'.
```